app/controllers/v1/sessions_controller.rb

```diff
@@ -243,6 +237,23 @@ def callback_stats(status, saml_response = nil, failure_tag = nil)
       end
     end
 
+    # rubocop:disable Metrics/ParameterLists
```

Comment:
This refactor drops the `      unless performed?` guard clause that was in the second rescue block. Are we confident that's no longer needed?